lib/pdk/tests/unit.rb

```diff
@@ -6,9 +6,13 @@
 module PDK
   module Test
     class Unit
-      def self.cmd(_tests)
+      def self.cmd(_tests, opts = {})
         # TODO: test selection
-        [File.join(PDK::Util.module_root, 'bin', 'rake'), 'spec']
+        if opts.key?(:parallel)
+          [File.join(PDK::Util.module_root, 'bin', 'rake'), 'parallel']
```

Comment:
As this is just deciding between two rake tasks, you can remove a bit of duplication here by changing the method content to something like this

```ruby
rake_task = opts.key?(:parallel) ? 'parallel' : 'spec'
[File.join(PDK::Util.module_root, 'bin', 'rake'), rake_task]
```